guides/wide-ep-lws/inferencepool-pd-override.values.yaml

```diff
@@ -0,0 +1,59 @@
+inferenceExtension:
```

Comment:
We should be able to reuse the entire pd values.yaml, and only need to override the metrics secret I believe. Can we get rid of this file and just do `set` in `helm apply`?